keyboards/aki27/cocot46plus/rules.mk

```diff
@@ -0,0 +1,4 @@
+SRC += matrix.c
+CUSTOM_MATRIX = lite
+POINTING_DEVICE_DRIVER = adns5050
+SPACE_CADET_ENABLE = no
```

Comment:
If required, this should be added to `keyboard.json` features block as `"space_cadet": false`
```suggestion
```

Reply:
Now that we are using RP2040, I'll just leave this enabled. Thanks for pointing out this detail.